core.go

```diff
@@ -188,25 +189,32 @@ func (c *core) addExceptionsFromError(
 	err error,
 ) []sentry.Exception {
 	for i := 0; i < maxErrorDepth && err != nil; i++ {
+		wrappedErr := err
+		err = errors.Cause(err)
 		if _, ok := processedErrors[getTypeOf(err)]; ok {
 			return exceptions
 		}
 
 		processedErrors[getTypeOf(err)] = struct{}{}
 
 		exception := sentry.Exception{Value: err.Error(), Type: reflect.TypeOf(err).String()}
+		if strings.HasSuffix(exception.Type, "errors.fundamental") {
+			// err was created with `errors.New(msg)` - make `msg` the type,
```

Comment:
I actually see `*errors.errString` here (see https://goplay.tools/snippet/txizx5D1JSQ).

Is the comment maybe not up-to-date? Or the check?

Also, this type doesn't seem to have ever been renamed since its [introduction 12 years ago](https://github.com/golang/go/commit/e4ae30f5f55b6b6ba2cb9a0c0e3b11d42dc162ee#diff-43a211a9a8f6f082210eaf94fb49a482cede5c59385a67e712b78ea530f24681R14), but it may still be a bit brittle to check directly against it. Maybe it would be enough to check for any error type inside of `errors`?